server/sonar-server/src/main/java/org/sonar/server/issue/IssueFieldsSetter.java

```diff
@@ -256,6 +256,13 @@ public void setCloseDate(DefaultIssue issue, @Nullable Date d, IssueChangeContex
     }
   }
 
+  public void setCreationDate(DefaultIssue issue, Date d, IssueChangeContext context) {
+    // TODO do we need date comparison, as in setCloseDate?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)